core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -3409,15 +3409,24 @@ private Tester getExtendedTester() {
     sql(sql).ok();
   }
 
-  @Test public void testWithinGroup1() {
+  @Test
```

Comment:
remove this test. you have not changed SqlToRelConverter.

Reply:
Done :)